lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,29 @@
+module ShopifyAPI
+  # avoid removing root_key as it activeresource object initialization
+  # https://github.com/rails/activeresource/blob/master/lib/active_resource/formats/json_format.rb#L21
+  module VaultSessionFormat
```

Comment:
I feel this should be in a generic class in a file on its own. It's used by, but not necessarily related to `VaultSession`.